web/src/test/auto/headless/engine/keyboard/deadkeys.js

```diff
@@ -4,8 +4,9 @@ import fs from 'fs';
 import { createRequire } from 'module';
 const require = createRequire(import.meta.url);
 
-import { KeyboardInterface, MinimalKeymanGlobal } from '@keymanapp/keyboard-processor';
-import { NodeKeyboardLoader } from '@keymanapp/keyboard-processor/node-keyboard-loader';
+import { MinimalKeymanGlobal } from 'keyman/engine/keyboard';
+import { KeyboardInterface } from 'keyman/engine/js-processor';
```

Comment:
As this references js-processor components, I'd suggest putting this within headless/engine/js-processor instead.